spec/ddtrace/contrib/delayed_job/plugin_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'spec_helper'
+require 'ddtrace/contrib/delayed_job/plugin'
+
+require_relative 'app'
+
+SampleJob = Struct.new('SampleJob') { def perform; end }
+
+RSpec.describe Datadog::Contrib::DelayedJob::Plugin do
+  let(:pin) { Datadog::Pin.get_from(::Delayed::Worker) }
+  let(:tracer) { ::Datadog::Tracer.new(writer: FauxWriter.new) }
+
+  before do
+    Datadog::Contrib::DelayedJob::Patcher.patch
```

Comment:
Why do we call `patch` directly instead of `Datadog.configure { |c| c.use :delayed_job }`? It would be best to preserve a consistent configuration API experience, whenever possible.